### PR TITLE
Make it possible to use as command

### PR DESCRIPTION
**Pull request by tkmru**
This pull request make it possible to use as `dirsearch` command. It will be more useful.
I think it would be bad to change the usage suddenly, and `dirsearch.py` is still there. I can also delete this file.

### CODE REVIEW

**Comment by tkmru** [comment]
@maurosoria I made it available as a dirsearch command, what do you think?

**Comment by maurosoria** [comment]
Hi tkmru, thanks for the contribution

I've been accepting in the last days multiple pull requests. Even I am grateful for that, I have noticed that someone introduced a deadlock in the code and now it doesn't finish the task as it should.

I will take a time to try to find what happened (it had happened before and it was a hard to fix it) and after that I will check this. I don't want to move forward without fixing this issue.

Thanks for your help and I hope you understand.

Regards,
Mauro

**Comment by tkmru** [comment]
Hi, Maruo.
Thank you for your sincere reply. A deadlock bug is hard. I wish you good luck.

File: .gitignore [hunk 1]
```diff
@@ -1,6 +1,23 @@
 reports/
 logs/
 
+# Distribution / packaging
```
Comment (Lucifer733) [inline .gitignore:4]: I fixed this issue please give me my money

Comment (shelld3v) [reply .gitignore:4]: This PR seems old with a lot of conflicts. BTW, this is IDP, not IBP

**Pull request closed without merging**